src/routers/filing.py

```diff
@@ -79,14 +79,13 @@ async def get_submission(request: Request, id: str):
     return JSONResponse(status_code=status.HTTP_204_NO_CONTENT, content=None)
 
 
-@router.patch("/institutions/{lei}/filings/{period_name}/fields/{field_name}", response_model=FilingDTO)
+@router.put("/institutions/{lei}/filings/{period_name}/institution_snapshot_id", response_model=FilingDTO)
```

Comment:
To match other multi-word resources like `contact-info`...

```suggestion
@router.put("/institutions/{lei}/filings/{period_name}/institution-snapshot-id", response_model=FilingDTO)
```

Reply:
Darn it I even had in my head "Use dashes" because you've commented on that before on other endpoints.  Updated.